blog/Website/2015-02-18_FreshRSS_gets_a_new_website.md

```diff
@@ -0,0 +1,6 @@
+Title: FreshRSS gets a new website
+Date: 2015-02-18
+
+We have worked a lot to improve community processes since last weeks. Today we are opening a new section on the website: a blog!
+
+TODO: this article needs to be finished!
```

Comment:
Ok, I'll let you remove it and push the modification.
I already reviewed the PR and it looks good to me.